src/test/java/com/connexta/search/query/GetFilterTests.java

```diff
@@ -39,4 +46,18 @@ public void testMalformedQuery() {
     assertThrows(
         MalformedQueryException.class, () -> QueryServiceImpl.getFilter("contents LIKE don't"));
   }
+
+  @Test
+  void testAcceptsAllValidQueryAttributes() {
+
+    // Assert that test data contains all valid terms.
+    Assert.assertThat(
+        "Sample data must include all query attributes",
+        unmodifiableSet(SearchTestData.map.keySet()),
+        equalTo(QUERY_TERMS));
+
+    // Assert all terms are accepted
+    Filter filter = QueryServiceImpl.getFilter(SearchTestData.allAttributesQuery);
```

Comment:
I deleted this class in my PR, as well as removed the getFilter method. I think this test is more appropriate to be tested through the query method. We care more about the query method accepting these attributes.

Reply:
Slide over and give me your recommendation.

Reply:
I'll move this test to go through the public method when @peterhuffer's code is merged and I rebase.